src/sbt-test/sbt-cake/simple/src/main/scala/Hello.scala

```diff
@@ -1 +1 @@
-object TerribleFormatting{def foo=   "bar"}
+object TerribleFormatting { def foo = "bar" }
```

Comment:
this was intentionally unformatted so we could test that the formatting is performed in the test.